impeller/entity/contents/filters/gaussian_blur_filter_contents.cc

```diff
@@ -481,11 +485,9 @@ fml::StatusOr<RenderTarget> MakeBlurSubpass(
                 linear_sampler_descriptor));
         GaussianBlurVertexShader::BindFrameInfo(
             pass, host_buffer.EmplaceUniform(frame_info));
-        GaussianBlurPipeline::FragmentShader::KernelSamples kernel_samples =
-            LerpHackKernelSamples(GenerateBlurInfo(blur_info));
-        FML_CHECK(kernel_samples.sample_count <= kGaussianBlurMaxKernelSize);
```

Comment:
Please bring back this check.  It's important because we'll crash anyways if we violate this.

Reply:
This is checked in the LerpHackFunction too, I don't think the count can change after that point?

Reply:
Okay yea, in that case I'd rather keep this one here since it is the final point before we'd actually get the error.  We can remove the lerp hack one.

Reply:
But the kernel samples UBO has a std::array of a fixed size. It cannot be greater than kGaussianBlurMaxKernelSize

Reply:
I could static assert this though.

Reply:
Ah yea, the `sample_count` isn't even consequential here because emplace isn't looking at it.  Lets keep the runtime DCHECK in the lerp hack and a static assert sounds good to me.

Reply:
Done